score-server/src/test/java/bio/overture/score/server/security/JWTSecurityTest.java

```diff
@@ -174,33 +204,33 @@ public void validateCanUploadWithValidAuthScopes() {
         return ResponseEntity.status(mvcResponse.getStatus()).body(responseObject);
     }
 
+    private final Map<ScopeOptions, List<String>> downloadScopesMap = Map.of(
+            VALID_SYSTEM,   List.of(resolveSystemDownloadScope(), "score.READ", "id.READ"),
+            VALID_STUDY,    List.of(resolveStudyDownloadScope(EXISTING_PROJECT_ID), "id.READ"),
+            INVALID_STUDY,  List.of(resolveStudyDownloadScope(NON_EXISTING_PROJECT_ID), "id.READ")
+    );
+
+    private final Map<ScopeOptions, List<String>> uploadScopesMap = Map.of(
+            VALID_SYSTEM, List.of(resolveSystemUploadScope(), "score.WRITE", "id.WRITE"),
```

Comment:
remove `score.WRITE` from this list. Even if the system upload scope will be `aws.WRITE`, its just misleading to have score.WRITE